examples/client/client.c

```diff
@@ -1360,7 +1363,7 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 #endif  /* !defined(NO_FILESYSTEM) */
     }
 
-    if (!usePsk && !useAnon) {
+    if (!usePsk && !useAnon && !useVerifyCb) {
```

Comment:
The new -j option works because the verify certs (CA's) are not loaded. So the myVerify callback is used and it overrides the failure and the client is able to connect with SSL_VERIFY_PEER set. The success of using the -j option indicates the callback was executed and the failure overridden.

I agree we should add another test for the date override and use expired certificate. I will put that on my test enhancement list to add in another PR.